Data/fragmentdata.txt

```diff
@@ -29,7 +29,7 @@
 	{
 		"Id":4,
 		"Title":"Research is going well",
-		"Description":"An L5 stage patient's dissection is required.\nHelp from the queen carrier is required.",
+		"Description":"An L5 stage patient's dissection is required.\nQueen carrier must be willing to participate.",
```

Comment:
```suggestion
		"Description":"An L5 stage patient's dissection is required.\nQueen carrier must be cooperative.",
```

Reply:
Not sure if the wording I used is what orian intended here.

Reply:
That one's the same as the one under, I meant "Cooperation from the queen carrier is required." Queen carrier isn't a name so it's weird to use it as such.

Reply:
ok, I changed it to the same as number 10, "Cooperation from the queen carrier is required."

Reply:
For reference, the phrasing used in the fragment is "Rika is willing to help"

https://github.com/07th-mod/matsuribayashi/blob/89ed4b2dd1c425ebca24889979de4a3eba3f31de/Update/_kakera07.txt#L829